### PR TITLE
ZipCode Analyzer

### LINKED ISSUE #8: Home address detection
It is important to identify and manage personal identifiable information (PII) . In this issue, I will  concentrate on identifying and safeguarding the Home address as the PII. A person's home address is  information that can be used for identity theft and other fraudulent acts.  Everyone should include safeguards out applications to recognize and protect this kind of PII.

For example:
xxxx Cooper lane, city name, state, zip code.

Detection Steps:
Python code will be developed which will compare two strings for an exact match. This code will check to if the address in string1 matches the address of string2. If the results matches the program will output true; If the the results do not match then the output will be false.

### LINKED ISSUE #37: Zip code anonymized
zip code is not detected and need to be worked on. I was the one assigned this PII but I have no clue how to get it detected. any assistance would help.

### PR DESCRIPTION
**Pull request by Natep23**
Added Zip Code Detection

### CODE REVIEW

File: pii_scan.py
Comment (claesmk) [inline pii_scan.py:48]: I think you have an extra space in your regex - should be no space between the | and the (
`(\b\d{5}(?!-)\b)|(\b\d{5}-\d{4}\b)`

If you are going to try to catch extended zip codes, ie 12345-1234 you need to include a test for that in your test case as well

**Review by claesmk — CHANGES_REQUESTED** [review]

File: pii_scan.py
Comment (Natep23) [reply pii_scan.py:48]: The Changes have been updated. Thanks

**Review by claesmk — APPROVED** [review]

**Comment by claesmk** [comment]
@Natep23 once you get one more review you should be able to merge this.  I assume you'll want to link #8 and #37 to this and also close #28

**Comment by claesmk** [comment]
@hsalaam this is waiting on your code review

**Review by brmtalla — APPROVED** [review]

**Pull request merged**
